### PR TITLE
Develop to Master

### PR DESCRIPTION
**Pull request by reynolkb**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -65,47 +128,49 @@
...
+#itinerary-title {
+    background-color: #ffbe76;
+    color: white;
+    font-weight: 400;
+    font-size: 20px;
 }
 
-.invalid-search {
-    border: solid red;
-    display: block;
```
Comment (ginomaglaqueucla) [inline assets/css/style.css:78]: Won't this remove the red error handling for invalid city?

Comment (reynolkb) [reply assets/css/style.css:78]: This code is still in there, I just moved it

Comment (ginomaglaqueucla) [reply assets/css/style.css:78]: sorry I didn't catch it earlier

**Review by ginomaglaqueucla — APPROVED** [review]

**Comment by reynolkb** [comment]
need to add readme

**Pull request closed without merging**